config/jobs/periodic/housekeeping/main.go

```diff
@@ -0,0 +1,62 @@
+package main
```

Comment:
can we have a separate project altogether for this task for better maintainability?

Reply:
https://github.com/ppc64le-cloud/gcs-cleaner.git here is the repository created.